src/main/webapp/mentors.js

```diff
@@ -2,10 +2,10 @@
  * Get mentors from the mentor servlet.
  */
 function getMentors() {
-  console.log("entering get mentors function/n");
+  console.log('entering get mentors function/n');
   let url = new URL('/mentors', location.protocol + '//' + location.hostname);
```

Comment:
Q: Are you using the URL object here for a particular reason? I'm not sure how this differs from
```javascript
fetch('/mentors')
```
Also, when I ran this code in the demo branch, this wasn't working. It only worked when I added the endpoint directly to the fetch call as shown in the snippet above.

Reply:
I was using a url object in case we want to add search params later. And, just to clarify, it only ran for you when you used '/mentors' instead of the URL object?

Reply:
Okay that makes sense for using URL. And yes, that clarification is correct. Let's look into it during our sync.